src/test/java/de/hpi/bpt/argos/api/EntityTypeEndpointTest.java

```diff
@@ -82,11 +89,12 @@ public void testGetAttributes() {
     }
 
     @Test
-    public void testGetAttributes_InvalidId_BadRequest() {
+    public void testGetAttributes_InvalidId_Success() {
         RestRequest request = RestRequestFactoryImpl.getInstance()
                 .createGetRequest(ARGOS_REST_HOST, EntityTypeEndpoint.getEntityTypeAttributesUri(root1.getId() - 1));
 
-        assertEquals(HttpStatusCodes.BAD_REQUEST, request.getResponseCode());
+        assertEquals(HttpStatusCodes.SUCCESS, request.getResponseCode());

```

Comment:
Is that another thing that would be really ugly otherwise or why did you change that?